<robot name="hole_0001">
  <link name="world"></link>

  <joint name="base_joint" type="fixed">
  <origin xyz="0 0 0" rpy="0 0 0"/>
  <parent link="world"/>
  <child link="hole_0001"/>
  </joint>

  <link name="hole_0001">
    <inertial>
      <origin xyz="0.0 0.0 0.035" rpy="1.570759 0 0"/>
      <mass value="0.228" />
      <inertia  ixx="0.0004921003" ixy="0.0"  ixz="0.0"  iyy="0.0002367729"  iyz="0.0"  izz="0.0004921003" />
    </inertial>
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="1.570759 0 0"/>
	<geometry>
		<mesh filename="package://sawyer_sim_examples/models/peg_hole/STL/rigid_hole_0001.STL" scale="0.001 0.001 0.001"/>
	</geometry>
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="1.570759 0 0"/>
	<geometry>
		<mesh filename="package://sawyer_sim_examples/models/peg_hole/STL/rigid_hole_0001.STL" scale="0.001 0.001 0.001"/>
	</geometry>
    </collision>
  </link>
  <gazebo reference="hole_0001">
    <material>Gazebo/Blue</material>
    <static>true</static>
     <mu1>10.0</mu1>
     <mu2>10.0</mu2>
  </gazebo>
</robot>
